### DISCUSSION TITLE
Create Custom Backend Info Pages

### DISCUSSION THREAD

**Discussion by olivdee** [joomla/joomla-cms#40177] [2. Cold Features]
I was wondering if there is an easy way in Joomla 4, to create an article (page) that shows up somehow only in the backend. Like some kind of an info page. The background on this is, that I want to provide the client of a project information on some modules, components and other project related stuff, right in the backend - something like a backend "Wiki"- or "Onboarding Page". 

I am aware of the feature of creating a backend custom modules (let's say/call it "Project Info"), add first short info in there and show it on the dashboard. But it would be handy, if one could place a link inside it, opening another page/article in the backend, with further/in-depth information on a chosen subject.

I couldn't find anything on this online, and also I did not find a solution while experimenting on the backend, so I assume it is not possible? Or maybe it is, in this case a hint to the right direction would be very welcome!

Thanks in advance for any help.

**Comment by micker** [comment]
you can do it via 2 things
- use dashboard + custom html module in good position
- you can test this module : https://github.com/micker/mod_dashboard
